src/sass/partials/base/_base.scss

```diff
@@ -0,0 +1,312 @@
+    margin-left: 0;
+    padding-left: 0;
+  }
+
+  &--ol {
+    .base--li {
+      padding-left: .2em;
+
+      &::before {
+        display: none;
+      }
+    }
+  }
+
+  &--hr {
+    border: 0 none;
+    border-top: 2px solid color('grey', 20);
+    margin: setting-get('rhythm') / .25;
+    margin-left: 0;
+    margin-right: 0;
+    width: 15%;
+  }
+
+  &--table {
+    width: 100%;
+  }
+
+  &--caption {
+    color: color('gray', 50);
+    font-size: .8em;
+    padding: setting-get('rhythm') / 2;
+    padding-left: 0;
+    padding-right: 0;
+    padding-top: 0;
+    text-align: right;
+  }
+
+  &--thead {
+
+    color: setting-get('white');
+  }
+
+  &--tr {
+    &:nth-of-type(even) {
+      background-color: color('neutral-white', 30);
+
+      & .base--td::before {
+        background-color: color('teal', 60);
+        // responsive table zebra striping
+      }
+    }
+  }
+
+  &--th {
+    display: none;
+    font-size: .8em;
+    font-weight: setting-get('normal font weight');
+    padding: setting-get('rhythm');
+    text-align: left;
+    @media (min-width: 600px) {
+      background-color: setting-get('primary color');
+      display: table-cell;
+      font-size: 1em;
+    }
+  }
+
+  &--td {
+    display: flex;
+    font-size: .8em;
+    margin: 0;
+    padding: setting-get('rhythm') / 2;
+    @media (min-width: 600px) {
+      display: table-cell;
+      font-size: 1em;
+      padding: setting-get('rhythm');
+    }
+
+    &::before {
+      @include hyphens(auto);
+      background-color: setting-get('primary color');
+      color: setting-get('white');
+      content: attr(data-th)': ';
+      display: inline-block;
+      margin: -.5rem .5em -.5rem -.5rem;
+      min-width: 6.5em;
+      padding: setting-get('rhythm') / 2;
+      text-align: right;
+      width: 6.5em;
+      word-break: break-all;
+
+      @media (min-width: 600px) {
+        display: none;
+      }
+    }
+  }
+
+  &--kbd {
+    background-color: color('neutral-white', 40);
+    border: 1px solid color('gray', 10);
+    border-radius: 3px;
+    font-size: .8em;
+    padding: .2em .5em;
+  }
+
+  &--code {
+    background-color: setting-get('black');
+    color: setting-get('secondary accent color');
+    font-size: .8em;
+    padding: .4em .5em;
+  }
+
+  &--samp {
+    color: setting-get('secondary color');
+  }
+
+  &--pre {
+    direction: ltr;
+    display: block;
+    hyphens: none;
+    overflow-y: overlay;
+    tab-size: 2;
+    text-align: left;
+    white-space: pre;
+    word-break: normal;
+    word-spacing: normal;
+    word-wrap: normal;
+  }
+
+  &--em {
+    font-weight: setting-get('bold font weight');
+  }
+
+  &--b {
+    font-weight: setting-get('bold font weight');
+  }
+
+  &--i {
+    font-style: italic;
+  }
+
+  &--u {
+    border-bottom: 2px solid setting-get('text color');
+    text-decoration: none;
+  }
+
+  &--strong {
+    font-weight: setting-get('bold font weight');
+  }
+
+  &--sup {
+    font-size: .8em;
```

Comment:
I see this `.8em` font size popping up a lot. Maybe worthwhile making it a setting?